docker-server.Dockerfile

```diff
@@ -2,52 +2,16 @@
 # See the DOCKER.md file in the root of the project for more information.
 # Please run `docker compose up` from the root of the project to run the docker environment for
 # the UCW-APP project, which this Dockerfile is part of.
-ARG WRKDR=/opt/app
+FROM node:20
 
-FROM alpine:3.20.3 AS base
-ENV NODE_VERSION 20.15.0
+WORKDIR /usr/src/app
 
-RUN apk --update --no-cache --virtual add nodejs npm \
-    && rm -rf /var/cache/apk/*
-
-FROM base AS pruner
-RUN npm i -g turbo
-ARG APP
-ARG WRKDR
-
-WORKDIR ${WRKDR}
-
-COPY . ${WRKDR}
-RUN turbo prune --scope=${APP} --docker
-
-FROM base AS builder
-ARG APP
-ARG WRKDR
-
-WORKDIR ${WRKDR}
-
-COPY --from=pruner ${WRKDR}/out/json/apps/${APP}/package.json .
-COPY --from=pruner ${WRKDR}/out/json/packages/utils/package.json ./packages/utils
-COPY --from=pruner ${WRKDR}/out/package-lock.json .
-
-RUN npm pkg delete scripts.prepare  \
-    && npm ci --omit=dev
-
-FROM base AS runner
-ARG APP
-ARG WRKDR
-
-WORKDIR ${WRKDR}
-
-RUN npm i -g ts-node  \
-    && addgroup --system --gid 1001 nodejs \
-    && adduser --system --uid 1001 nodejs
-USER nodejs
-
-COPY --from=pruner --chown=nodejs:nodejs ${WRKDR}/out/full/apps/${APP}/ .
-COPY --from=pruner --chown=nodejs:nodejs ${WRKDR}/out/full/packages/utils/ ./packages/utils
-COPY --from=builder --chown=nodejs:nodejs ${WRKDR}/node_modules/ ./node_modules
+# Copy app source
+COPY . .
+ 
+RUN npm install
+RUN npm run build
 
 EXPOSE ${PORT}
 
-CMD ["ts-node", "./src/server.js"]
+CMD ["npm", "run", "start:server"]
```

Comment:
You're losing a lot of security in the docker file by removing the code that created a limited user, and instead just running everything as root.

Also, moving from `alpine` to `node` image will make the final docker image much larger. The original Dockerfile that was created by sophtron resulted in an image that was over 1GB is size.  After moving to a slimmer image base, and using different build contexts, the image was around 180MB.